### DISCUSSION TITLE
How to have both old and new versions in a single (large) project?

### DISCUSSION THREAD

**Discussion by sknutsonsf** [Comcast/react-data-grid#3456] [Q&A]
Is there a way I can have RDG 7 installed in the same project with RDG 6 using Yarn V2?
I'm adding new module and I'd like to use the newer library for it.
Its the same "npm module name" which is the problem

Yarn V2 does not have aliasing like yarn v1 did.

We have more 15 existing RDG 6 lists, some of which are very complex.  It is not feasible to rewrite those to RDG7 anytime soon.
I'm not sure we will ever justify the level of rewrite needed.

(or release on NPM as "rdg7" or something to make it easy)

**Comment by nstepien** [comment]
I don't how to do it in yarn, but with npm you can run
```
npm i react-data-grid-next@npm:react-data-grid@latest
```
